modules/container-base/README.md

```diff
@@ -0,0 +1,56 @@
+# Dataverse Base Container Image
+
+A "base image" offers you a pre-installed and pre-tuned application server to deploy Dataverse software to.
+Adding basic functionality like executing scripts at container boot, monitoring, memory tweaks, etc., is all done
+at this layer, to make the application image focus on the app itself.
+
+## Quick Reference
+
+**Maintained by:** 
+
+This image is created, maintained and supported by the Dataverse community on a best-effort basis.
+
+**Where to find documentation:**
+
+The [Dataverse Container Guide - Base Image](https://guides.dataverse.org/en/latest/container/base-image.html)
+provides in-depth information about content, building, tuning and so on for this image. 
+
+**Where to get help and ask questions:**
+
+IQSS will not offer you support how to deploy or run it. Please reach out to the community for help on using it.
+You can join the Community Chat on Matrix at https://chat.dataverse.org or the Community Slack at
+https://dataversecommunity.slack.com to ask for help and guidance.
+
+## Supported Image Tags
+
+This image is sourced within the main upstream code [repository of the Dataverse software](https://github.com/IQSS/dataverse).
+Development and maintenance happens there (again, by the community). Community-supported image tags are based on the two
```

Comment:
By now the reader is quite clear who is supporting this thing... the community! 😄 I'm ok with this fact being over-communicated. 😄